### PR TITLE
Ability to test pure SWT apps without adding dependencies on eclipse

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Comment by rawagner** [comment]
*add missing copyright for new files

**Comment by rawagner** [comment]
testPR

File: plugins/org.jboss.reddeer.workbench.core/about.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<head>
+<title>About</title>
+<meta http-equiv=Content-Type content="text/html; charset=ISO-8859-1">
+</head>
+
+<BODY lang="EN-US">
+
+<H3>About This Content</H3>
+
+<P>&copy;2010 Red Hat, Inc. All rights reserved</P>
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.workbench.core/about.html]: 2016?

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/WidgetIsDisposed.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.jboss.reddeer.core.condition;
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/WidgetIsDisposed.java]: add license info please

File: plugins/org.jboss.reddeer.workbench.core/src/org/jboss/reddeer/workbench/core/lookup/ViewToolBarLookup.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package org.jboss.reddeer.workbench.core.lookup;
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.workbench.core/src/org/jboss/reddeer/workbench/core/lookup/ViewToolBarLookup.java]: add license info

File: plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/handler/WorkbenchShellHandler.java [hunk 4]
```diff
@@ -0,0 +1,155 @@
+package org.jboss.reddeer.workbench.handler;
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/handler/WorkbenchShellHandler.java]: add license info please

File: plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/lookup/ViewMenuLookup.java [hunk 5]
```diff
@@ -0,0 +1,198 @@
+package org.jboss.reddeer.workbench.lookup;
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/lookup/ViewMenuLookup.java]: add license info please

File: tests/org.jboss.reddeer.jface.test/src/org/jboss/reddeer/jface/test/viewer/PrepareTreeWithStyledItems.java [hunk 6]
```diff
@@ -68,16 +68,16 @@ public void run() {
...
 			public void run() {
-				for (Shell shell : org.jboss.reddeer.core.util.Display.getDisplay().getShells()) {
+				for (Shell shell : org.jboss.reddeer.common.util.Display.getDisplay().getShells()) {
 					if (shell.getText().equals(title)) {
 						shell.dispose();
 						break;
 					}
 				}
-				new WaitWhile(new ShellWithTextIsActive(title));
+				new WaitWhile(new ShellWithTextIsAvailable(title));
```
Comment (mlabuda) [inline tests/org.jboss.reddeer.jface.test/src/org/jboss/reddeer/jface/test/viewer/PrepareTreeWithStyledItems.java]: you have here waiting inside UI thread. It was done this way already but we can fix it, could you look at it?

File: plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/lookup/ViewMenuLookup.java (hunk 5 above)
Comment (rawagner) [inline plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/lookup/ViewMenuLookup.java]: fixed

File: plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/handler/WorkbenchShellHandler.java (hunk 4 above)
Comment (rawagner) [inline plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/handler/WorkbenchShellHandler.java]: fixed

File: plugins/org.jboss.reddeer.workbench.core/src/org/jboss/reddeer/workbench/core/lookup/ViewToolBarLookup.java (hunk 3 above)
Comment (rawagner) [inline plugins/org.jboss.reddeer.workbench.core/src/org/jboss/reddeer/workbench/core/lookup/ViewToolBarLookup.java]: fixed

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/WidgetIsDisposed.java (hunk 2 above)
Comment (rawagner) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/WidgetIsDisposed.java]: fixed

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/TableHandler.java [hunk 7]
```diff
@@ -216,18 +216,17 @@ public void run() {
...
 			public void run() {
-				TableItemHandler.getInstance().select(tableItem);
 				Rectangle rectangle = tableItem.getBounds(column);
 				int x = rectangle.x + (rectangle.width / 2);
 				int y = rectangle.y + (rectangle.height / 2);
 				WidgetHandler.getInstance().notifyItemMouse(
 						SWT.MouseDown, SWT.NONE, tableItem.getParent(),
 						tableItem, x, y, 1);
-				AbstractWait.sleep(TimePeriod.getCustom(1));
 				WidgetHandler.getInstance().notifyItemMouse(
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/TableHandler.java]: We notify item about MouseDown then MouseUp..and yes it is still needed

**Pull request merged**